content/en/news/releases/1.5.x/announcing-1.5/change-notes/index.md

```diff
@@ -4,4 +4,89 @@ description: Istio 1.5 release notes.
 weight: 10
 ---
 
-TBD
+## Traffic management
+
+- **Improved** performance of `ServiceEntry` updates #19305.
+- **Improved** fix readiness probe inconsistency #18164.
+- **Improved** performance of configuration updates by sending partial updates where possible #18354.
+- **Added** an option to control locality load balancer settings per host #18406.
+- **Fixed** an issue where pods crashing would trigger excessive configuration pushes #18574.
+- **Fixed** issues with applications that call themselves #19308.
+- **Added** detection of `iptables` failure when using Istio CNI #19534.
+- **Added** `consecutive_5xx` and `gateway_errors` as outlier detection options #19771.
+- **Improved** `EnvoyFilter` matching performance optimization #19786.
+- **Added** support for `HTTP_PROXY` protocol #19919.
+- **Improved** `iptables` setup to use `iptables-restore` by default #18847.
+- **Enabled** [automatic protocol detection](https://istio.io/docs/ops/configuration/traffic-management/protocol-selection/#automatic-protocol-selection-experimental) by default #18666.
+
+## Security
+
+- **Added** Beta authentication API. The new API separates peer (i.e mutual TLS) and origin (JWT) authentication into `PeerAuthentication` and `RequestAuthentication` respectively. Both new APIs are workload-oriented, as opposed to service-oriented in alpha `AuthenticationPolicy`.
+- **Added** Authorization Policy supports deny action and exclusion semantics.
+- **Added** auto mutual TLS. This beta feature is enabled by default.
+- **Improved** node agent by merging with Pilot agent. This improves the security posture by removing the requirement of a pod security policy.
+- **Improved** Istio by including certificate provisioning functionality within Pilot.
+- **Improved** Support Kubernetes [`first-party-jwt`](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#service-account-tokens) as a fallback token for CSR authentication in clusters that [`third-party-jwt`](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#service-account-token-volume-projection) is not supported.
+- **Improved** Istio's Pilot Agent by provisioning key and certificates for Prometheus consumption.
```

Comment:
pilot agent ->istio agent

Reply:
Can you change this item to the following?
- **Added** Istio Agent provisions a key and certificates for Prometheus.

Reply:
```suggestion
- **Added** Istio Agent provisions a key and certificates for Prometheus.
```